### ISSUE TITLE
Cover art - Art Uploading: Modifications to 'Art pin query' aren't saved upon foobar2000 restart

### ISSUE THREAD

**Issue by coderheck** [TheQwertiest/foo_discord_rich#77]
Description and repro steps is in the title
Component version: v2.0.2
OS: Windows 10

**Comment by xiiyuki** [comment]
Same here, latest dev build from [here](https://ci.appveyor.com/api/projects/theqwertiest/foo-discord-rich/artifacts/_result%2Fx64_Release%2Ffoo_discord_rich-x64.fb2k-component?branch=master&pr=false&job=Configuration%3A+Release%3B+Platform%3A+x64)

Even making the config file read-only doesn't enforce the plugin to read what's in the config. Or at least, partially: `Art pin query` always reverts to %artist%|%album%, but I've been able to sucessfully keep custom text fields in the main tab (Top being `$meta(title,0)`, and Middle being `$if($meta(album artist),$meta(album artist,0),$if($meta(artist),$meta(artist,0),))`. It seems like everything except `Art pin query` gets saved properly

**Comment by netceil** [comment]
same as #66, was fixed in 44dbbe6

**Comment by xiiyuki** [comment]
> same as #66, was fixed in [44dbbe6](https://github.com/TheQwertiest/foo_discord_rich/commit/44dbbe6178ce44ac1c8692db9883a19aab175eb8)

It seems that this commit happened after compiling of the latest dev build of the plugin, or it didn't get fixed if it was compiled with that commit.

**Comment by netceil** [comment]
> It seems that this commit happened after compiling of the latest dev build of the plugin, or it didn't get fixed if it was compiled with that commit.

nope, check if youre running the same dev build version as me, which is `v2.0.3-dev+44dbbe6`. the art pin query thing works, and i have succesfully changed it to `%album artist%|%album%`.

**Comment by xiiyuki** [comment]
> nope, check if youre running the same dev build version as me, which is `v2.0.3-dev+44dbbe6`. the art pin query thing works, and i have succesfully changed it to `%album artist%|%album%`.

I'm using the exact mentioned version and again, there's no issue with not being able to modify "Art pin query". The issue is that it's not possible to keep the modified entry after foobar2000 is restarted. Everytime the value for "Art pin query" returns to `%artist%|%album%`

**Comment by netceil** [comment]
> The issue is that it's not possible to keep the modified entry after foobar2000 is restarted. Everytime the value for "Art pin query" returns to %artist%|%album%

it works for me, even after a restart
